cli/Sources/TuistLoader/Models+ManifestMappers/TuistGeneratedProject+ManifestMapper.swift

```diff
@@ -15,10 +15,19 @@ extension TuistCore.TuistGeneratedProjectOptions.GenerationOptions {
                 return nil
             }
         }()
+        var additionalPackageResolutionArguments = manifest.additionalPackageResolutionArguments
+        if manifest.resolveDependenciesWithSystemScm {
+            additionalPackageResolutionArguments.append("-resolvePackageDependenciesWithSystemScm")
+        }
+        if let clonedSourcePackagesDirPath {
+            additionalPackageResolutionArguments.append("-clonedSourcePackagesDirPath")
+            additionalPackageResolutionArguments.append(clonedSourcePackagesDirPath.pathString)
```

Comment:
Note [here](https://github.com/tuist/tuist/pull/8099/files#diff-78608e5863f655199c6eb776080d19af574887430dd7bca77a9a1d8d830b9357L74) that the path that we pass is a concatenation of `clonedSourcePackagesDirPath` and `workspacePath`, so we'll have to keep the same logic here.